### ISSUE TITLE
Quokka webview broken in vscode: Error loading webview: Error: Could not register service workers: 

### ISSUE THREAD

**Issue by apolopena** [wallabyjs/quokka#646]
### Issue description or question

I can't do anything in vscode with the quokka plugin that uses the webview. The error `Error loading webview: Error: Could not register service workers: InvalidStateError: Failed to register a ServiceWorker: The document is in an invalid state` pops up and the webview for quokka is blank. **_I can't manage my license key_**. I can get the quokka plugin to work but due to this issue I cannot use the pro version I paid for.

```javascript
// paste the code you are running here
// n/a
```
![image](https://user-images.githubusercontent.com/3060702/120240451-70f2e380-c215-11eb-98dd-1254146ccfcb.png)

### Sample repository link

n/a

### Quokka.js Console Output

```
// paste your quokka console output here
```

### Code editor version

Visual Studio Code v1.5.62
Quokka plugin version: 1.0.371

### OS name and version

Windows 10.0.19042.0

**Comment by smcenlly** [comment]
This error is coming from VS Code and not Quokka. Quokka itself (including the start page) does not use any ServiceWorkers for any of its code/dependency code.

In VS Code, after you see this error, can you please run the command, `Developer: Toggle Developer Tools` and provide us with the `console.log` as it should have some additional details that may help.

We found this open VS Code issue repo from 27 days ago: [microsoft/vscode#122951- Can't open any webviews](https://github.com/microsoft/vscode/issues/122951) where a number of people have reported the same problem that you have. Reinstalling VS Code fixed the problem for some people, although you may need to delete VS Code AppData folder.

To set your license key without using the start page, please create a `.quokka` folder under your home directory if it does not exist. Within that folder, please create a `.qlc` file (~\.quokka\.qlc`) with the contents of your license key (no line breaks).

**Comment by smcenlly** [comment]
We're going to assume that our suggestion worked for you. If not, please let us know and we can re-open this issue if required.

**Comment by apolopena** [comment]
Only the Quokka webview triggers this issue for some reason. I will use the workaround you gave me for now.

**Comment by smcenlly** [comment]
> Is there a way for me to disable the "Quokka: What's New" webview auto popup? Those popups literally stack up on each other if I don't close them every time I open vscode.

Yes. If it doesn't already exist, create a folder in your home directory called `.quokka`. Under that folder, create a file called `startPage.json` with the contents as shown below:

**~/.quokka/startPage.json**
```json
{"version":1}
```

---

We would really like to get to the bottom of your problem.  Did you try  a full reinstall (including deleting your VS Code AppData)? Could you please also provide us with the `console.log` output:

> In VS Code, after you see this error, can you please run the command, Developer: Toggle Developer Tools and provide us with the console.log as it should have some additional details that may help.

**Comment by apolopena** [comment]
@smcenlly Thank you for the follow up. I was reluctant to delete my `%AppData%/Code` folder because I wasn't sure  exactly what settings I would loose and I have a pretty complex vscode setup including wsl2 remote stuff and alot of customizations. Long story short, I did the following which fixed my issue, brought back the wallaby tools panel and kep all my VSCode settings intact. 

1. Backed up my `%AppData%/Code` folder
2. Uninstalled VSCode
2. Rebooted
3. Deleted my `%AppData%/Code` folder
4. Installed VSCode
5. Removed the `ServiceWorker` folder from my backup of `%AppData%/Code`
6. Copied the contents of my `%AppData%/Code` backup to `%AppData%/Code`
